### ISSUE TITLE
2021-11-9

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#650]
311Change navigation_tag == 1 to navigation_tag == “words”; done; change navigation_tag == 2 to navigation_tag == “sentences”; done; change navigation_tag == 3 to navigation_tag == “word_editor”; done; change navigation_tag == 4 to navigation_tag == “sentence_editor”; done; change navigation_tag == 5 to navigation_tag == “analyzer”; done; change navigation_tag == 6 to navigation_tag == “homepage”; done; change navigation_tag == 7 to navigation_tag == “phrases”; done; change position of phrases; done; (2021-10-28)
312A Simple Framework for Contrastive Learning of Visual Representations
313Ting Chen, Simon Kornblith, Mohammad Norouzi, Geoffrey Hinton
3142020, cited by  2380; 
315Abstract
316SimCLR: a simple framework for contrastive learning of visual representations; simplify recently proposed contrastive self-supervised learning algorithms; understand what enables the contrastive
317prediction tasks to learn useful representations; (1) composition of data augmentations plays a critical role; (2) nonlinear transformation between the representation and the contrastive loss; (3) contrastive learning benefits from larger batch sizes and more training steps; (2021-10-25)
318I.Introduction
3191.Learning effective visual representations without human supervision; generative or discriminative; Generative approaches learn to generate; pixel-level generation may not be necessary; Discriminative approaches similar to supervised learning; (2021-10-25)
